tests/dir.props

```diff
@@ -69,15 +69,20 @@
   <Import Project="$(RoslynPropsFile)" Condition="'$(RunningOnUnix)'!='true' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
   <Import Project="$(RoslynPropsFile)" Condition="'$(RunningOnUnix)'=='true' and Exists('$(RoslynPropsFile)')" />
 
+  <PropertyGroup>
+    <RestoreSources Condition="'$(OverwriteCoreClrPackageVersion)' == 'true'">
+      $(PackagesDir)AzureTransfer\;
+      $(RestoreSources)
+    </RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://api.nuget.org/v3/index.json;
+      $(OverridePackageSource);
```

Comment:
Do we need `$(OverridePackageSource)`?

Reply:
No, I'll remove--leftover from copying from CoreFX.

Reply:
While I don't want to change this here I wish they used OverridePackageSource instead hardcoding the AzureTransfer path behind the OverwriteCoreCLRPackageVersion property. @dagood would you mind filing an issue for that to make it consistent?

Reply:
Sure, created https://github.com/dotnet/coreclr/issues/14525.